etc/kayobe/kolla/config/prometheus/rabbitmq.rules

```diff
@@ -5,69 +5,76 @@
 groups:
 - name: rabbitmq.rules
   rules:
-  - alert: RabbitmqNodeDown
-    expr: sum(rabbitmq_build_info{host_name!=""}) < 3
+  - alert: RabbitMQNodeDown
+    expr: sum(rabbitmq_build_info{instance!=""}) < 3
     for: 30m
     labels:
       severity: critical
     annotations:
-      description: Rabbitmq node down on {{ $labels.host_name }}
-  - alert: RabbitmqConsumersLowUtilization
+      description: RabbitMQ node down on {{ $labels.instance }}
+  - alert: RabbitMQConsumersLowUtilization
     expr: rabbitmq_queue_consumer_utilisation < 0.4
     for: 5m
     labels:
       severity: warning
     annotations:
-      description: RabbitMQ consumers message consumption speed is low on {{ $labels.host_name }}
-  - alert: RabbitmqNodeNotDistributed
+      description: RabbitMQ consumers message consumption speed is low on {{ $labels.instance }}
+  - alert: RabbitMQNodeNotDistributed
     expr: erlang_vm_dist_node_state < 3
     for: 5m
     labels:
       severity: critical
     annotations:
-      description: Rabbitmq node not distributed on node {{ $labels.host_name }}
-  - alert: RabbitmqMemoryHigh
+      description: RabbitMQ node not distributed on node {{ $labels.instance }}
+  - alert: RabbitMQMemoryHigh
     expr: rabbitmq_process_resident_memory_bytes / rabbitmq_resident_memory_limit_bytes
       * 100 > 90
     for: 2m
     labels:
       severity: warning
     annotations:
-      description: Rabbitmq memory too high on {{ $labels.host_name }}
-  - alert: RabbitmqFileDescriptorsUsage
+      description: RabbitMQ memory too high on {{ $labels.instance }}
+  - alert: RabbitMQFileDescriptorsUsage
     expr: rabbitmq_process_open_fds / rabbitmq_process_max_fds * 100 > 90
     for: 2m
     labels:
       severity: warning
     annotations:
-      description: Rabbitmq file descriptors usage on {{ $labels.host_name }}
-  - alert: RabbitmqTooMuchUnack
-    expr: sum(rabbitmq_queue_messages_unacked) BY (queue) > 1000
+      description: RabbitMQ file descriptors usage on {{ $labels.instance }}
+  - alert: RabbitMQTooMuchReady
+    expr: rabbitmq_queue_messages_ready > 100
     for: 1m
     labels:
       severity: warning
     annotations:
-      description: Rabbitmq too much unack on {{ $labels.host_name }}
-  - alert: RabbitmqTooMuchConnections
+      description: RabbitMQ too much ready on {{ $labels.instance }}
+  - alert: RabbitMQTooMuchUnack
+    expr: rabbitmq_queue_messages_unacked > 100
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      description: RabbitMQ too much unack on {{ $labels.instance }}
+  - alert: RabbitMQTooMuchConnections
     expr: rabbitmq_connections > 1000
     for: 2m
     labels:
       severity: warning
     annotations:
-      description: Rabbitmq too much connections on {{ $labels.host_name }}
-  - alert: RabbitmqNoQueueConsumer
+      description: RabbitMQ too much connections on {{ $labels.instance }}
```

Comment:
minor nit: `much` should swapped for `many` for better grammar

Reply:
This will fix upstream, I will propose a sync.